root/edit/components/EditSidebar.js

```diff
@@ -75,7 +72,7 @@ const EditSidebar = ({$c, edit}: Props) => (
         )}
       </SidebarProperty>
 
-      <SidebarProperty className="" label={l('Conditions:')}>
+      <SidebarProperty className="" label={l('If no votes cast:')}>
```

Comment:
New messages should not end with colon:
```suggestion
      <SidebarProperty className="" label={addColonText(l('If no votes cast'))}>
```